src/main/java/taub/citi/Stations.java

```diff
@@ -0,0 +1,110 @@
+package taub.citi;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+public class Stations
+{
+    public Data data;
+
+    public Stations(List<Station> stations)
+    {
+        data = new Data(stations);
+    }
+
+
+    public Station getStationStatus(String currStationId)
+    {
+
+        Station currStation = this.data.stations.get(0);
+        for (Station station : this.data.stations)
+        {
+            if (station.station_id.equals(currStationId))
+            {
+                currStation = station;
+                break;
+            }
+        }
+        return currStation;
+    }
+
+    // usage is to call the method from the list of stationInfos and pass in the stationstatuses
+    // (This method won't work in the opposite order)
+    public Stations combineStationInfo(List<Station> stationStatus, int retVal)
```

Comment:
Rename retval to something that actually describes what it is

Reply:
I fixed this. Is everything else good?